### PR TITLE
Fix setViewport & setScissor pixelRatio bug

### PR DESCRIPTION
**Pull request by 06wj**
The current setBiewport&setScissor is not multiplied by pixelRatio.

### CODE REVIEW

File: src/renderers/WebGLRenderer.js [hunk 1]
```diff
@@ -2675,7 +2677,7 @@ function WebGLRenderer( parameters ) {
 		state.scissor( _currentScissor );
 		state.setScissorTest( _currentScissorTest );
 
-		state.viewport( _currentViewport );
+		!window.xx && state.viewport( _currentViewport );
```
Comment (mrdoob) [inline src/renderers/WebGLRenderer.js]: Hmm... What's this test for?

**Comment by mrdoob** [comment]
Hmm, I wonder why the current examples that use `setViewport()` and `setScissor()` don't render incorrectly. Do you have a test case showing the bug?

File: src/renderers/WebGLRenderer.js (hunk 1 above)
Comment (06wj) [inline src/renderers/WebGLRenderer.js]: Sorry, i forgot remove debug code

**Comment by 06wj** [comment]
It just looks right. Because setViewport first sets viewport to the wrong value and then resets to the correct value in [this line](https://github.com/mrdoob/three.js/blob/dev/src/renderers/WebGLRenderer.js#L2661). 
It will call gl.viewport twice.

**Comment by mrdoob** [comment]
> Because setViewport first sets viewport to the wrong value and then resets to the correct value in [this line](https://github.com/mrdoob/three.js/blob/dev/src/renderers/WebGLRenderer.js#L2661). 
It will call gl.viewport twice.

Interesting...

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**